lib/npm.js

```diff
@@ -38,12 +40,12 @@ module.exports.install = (deps = [], opts = {}) => {
     cwd: opts.directory || process.cwd(),
     stdio: opts.silent ? 'ignore' : 'pipe'
   })
-}
+};
 
 const packageTypes = ['tag', 'git', 'version', 'range', 'file', 'directory', 'remote']
 
 module.exports.normalizePackageName = normalizePackageName
-async function normalizePackageName (name, opts = {}) {
+function normalizePackageName (name, opts = {}) {
```

Comment:
Hm, at one points I swear I remember this doing something async.  Guess I forgot to remove that, good catch.